notebooks/Tutorial1_A_basic_avatarization.py

```diff
@@ -49,7 +49,7 @@
 
 df = pd.read_csv("../fixtures/iris.csv")
 
-df
+df.head()
```

Comment:
tiny details but `df` gives you the size of the dataframe while `df.head()` does not